src/Kdyby/Curl/FileResponse.php

```diff
@@ -214,4 +214,15 @@ public static function prepareDownload(CurlWrapper $curl, $dir)
 		));
 	}
 
+
+
+	/**
+	 * @param string $url
+	 * @return string
+	 */
+	public static function sanitizeFileName($url)
```

Comment:
I don't this alias for sha1 has to be here. You could have just inlined it in the `prepareDownload` method.
Also, `md5()` should be faster.